py3status/modules/file_status.py

```diff
@@ -68,17 +111,40 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
-            color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
+        # init datas
+        paths = sorted([files for path in self.path for files in glob(path)])
+        paths_number = len(paths)
+
+        # get thresholds
+        if self.thresholds:
+            self.py3.threshold_get_color(paths_number, 'paths')
+
+        # format paths
+        if self.init['format_path']:
+            format_path = {}
+            format_path_separator = self.py3.safe_format(
+                self.format_path_separator)
+
+            for key in self.init['format_path']:
+                if key == 'basename':
+                    temps_paths = map(basename, paths)
+                elif key == 'fullpath':
+                    temps_paths = paths
+                else:
+                    continue
+                format_path[key] = self.py3.composite_join(
+                    format_path_separator, temps_paths)
+
+            format_path = self.py3.safe_format(self.format_path, format_path)
 
         response = {
-            'cached_until': self.py3.time_in(self.cache_timeout),
-            'full_text': self.py3.safe_format(self.format, {'icon': icon}),
-            'color': color
+            'cached_until':
+            self.py3.time_in(self.cache_timeout),
+            'full_text':
+            self.py3.safe_format(self.format, {
+                'paths': paths_number,
+                'format_path': format_path
+            })
```

Comment:
(Request) An attempt on creating consistency in all modules and could be a project's coding style if we were to document it... to make it more readability or such.
```python
        return {
            'cached_until': self.py3.time_in(self.cache_timeout),
            'full_text': self.py3.safe_format(
                self.format, {
                    'paths': count_paths,
                    'format_path': format_path
                }
            )
        }
```